.golangci.yml

```diff
@@ -0,0 +1,57 @@
+run:
+    deadline: 2m
+
+    skip-dirs:
+        - client
+
+linters:
+  disable-all: true
```

Comment:
What is the rationale behind only enabling certain linters? I mean if this repo would want to be compliant with the Go linter rules, this may be the time to fix them now.

Reply:
By default only a subset of linters are enabled. The rest need to be explicitly be enabled. See: https://golangci-lint.run/usage/quick-start/

Reply:
Also this way you can make sure that the expected linters are enabled even if the default set changed.